vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is an end-to-end test. Test will ssh to a vm and create a volume using docker cli.
+// After creating a volume, test will ssh to the ESX and verify the properties of the volume.
+// Properties being verified - capacity, disk-format and vm-attached field.
+
+// Test assumes that SSH cert has been setup to enable password-less login
+package e2e_test
+
+import (
+	"fmt"
+	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils"
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"time"
+)
+
+var (
+	esx              string = os.Getenv("ESX")
+	volSizes                = []string{"100MB"}
+	vms                     = []string{os.Getenv("VM1")}
+	formatTypes             = []string{"thin", "zeroedthick", "eagerzeroedthick"}
+	vmIp             string = ""
+	dockerVolmRmvCmd string = "docker volume rm "
+	dockerCliCheck   bool   = true
+)
+
+func TestMain(m *testing.M) {
+	retCode := m.Run()
+	os.Exit(retCode)
+}
+
+// clean-up function
+func teardownFunction() {
+	fmt.Println("*****clean-up***** on vm - ", vmIp)
+	TestUtil.ExecCmd(vmIp, dockerVolmRmvCmd)
+	fmt.Println("-----Clean-up finished - current time: ", time.Now())
+}
+
+/*
+Test steps:
```

Comment:
Is this PR addressing the steps mentioned at #824? If so please point to the step which performs "verify the size of volume using vmkfstools" otherwise update the PR by removing its reference.